Source/Blazorise/Components/Table/Table.razor.cs

```diff
@@ -140,15 +140,16 @@ private async ValueTask RecalculateResize()
         }
 
         /// <summary>
-        /// If table has <see cref="FixedHeader"/> enabled, it will scroll position to the provided pixels.
+        /// If table has <see cref="FixedHeader"/> enabled, it will scroll to the provided value.
         /// </summary>
-        /// <param name="pixels">Offset in pixels from the top of the table.</param>
+        /// <param name="scrollTo"><see cref="TableScrollToType"/></param>
+        /// <param name="value">Value to Scroll to according to <see cref="TableScrollToType"/></param>
         /// <returns>A task that represents the asynchronous operation.</returns>
-        public ValueTask FixedHeaderScrollTableTo( int pixels )
+        public ValueTask FixedHeaderScrollTableTo( TableScrollToType scrollTo, int value )
```

Comment:
Name it ScrollTo?

Does it make sense to have an enum for operation type? Maybe just have two diferent methods?

Reply:
ScrollToRow and ScrollToOffset?

Reply:
I was kinda divided between both ways, haha.
My though was: If new ways come up, it's easy to add a new one, instead of making a new method for each one.

But I can change it, yes, Would you rather have the two methods, then?

Reply:
Refactored into two methods.